docs/index.rst

```diff
@@ -71,6 +71,22 @@ This tool helps transform legacy configuration files templated with the atparse
 
 | :any:`CLI documentation with examples<cli_template_translate_examples>`
 
+File Moving
+^^^^^^^^^^^
+
+This tool helps users define the source and location of files to be copied or linked in the same UW YAML language used by UW drivers.
+
+| :any:`CLI documentation with examples<cli_file_copy_examples>`
+
+
+There is a video demonstration of the use of the ``uw file`` tool available via YouTube.
+
+.. raw:: html
+
+   <iframe width="560" height="315" src="https://www.youtube.com/embed/b2HXOlt-Ulw?si=rLWatBFu4mvNR65S" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Comment:
I don't have any suggestions (or see a problem), but I'm fairly ignorant on YouTube best practices.

Reply:
I guess I was mainly worried about how we handle all other links through the conf.py file. I have no clue if this link will still get checked.

Reply:
I think it will -- you should be able to temporarily hack a typo into it in your clone and see `make docs` fail. `config.py` is just for DRYing out common links; I think Sphinx does the link checking after rendering all the links based on their `conf.py` prefixes and suffixes (if any) supplied in the `.rst` files.

Reply:
Thanks for the reminder! I tested the test and am disheartened to learn that it did not catch the bad link in the embedded html.

I'd like to put this on the "known issues to fix" list, and move forward with this PR, if that works.

Reply:
Oh, sorry, I just now understood the significance of your original comment/question: this is an `<iframe>` element and so Sphinx is probably (and apparently, per your test) ignorant of the embedded URL.